From 322bc75f9e5620601dd8e0fe7f78c29fa1d6697f Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Tue, 6 Nov 2018 16:50:23 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808: fix up the pd_pcie qos node

add usb2 and usb3 qos node,
set qos_pcie\qos_usb2\qos_usb3 status = "disabled" by default.

Change-Id: I307be4138c04c3bea5ae779ed85a9a4d2420ad0a
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 17 ++++++++++++++++-
 1 file changed, 16 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 92d6a996d1d7..75d889c592f7 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -342,6 +342,19 @@
 	qos_pcie: qos@fe880000 {
 		compatible = "syscon";
 		reg = <0x0 0xfe880000 0x0 0x20>;
+		status = "disabled";
+	};
+
+	qos_usb2: qos@fe890000 {
+		compatible = "syscon";
+		reg = <0x0 0xfe890000 0x0 0x20>;
+		status = "disabled";
+	};
+
+	qos_usb3: qos@fe890080 {
+		compatible = "syscon";
+		reg = <0x0 0xfe890080 0x0 0x20>;
+		status = "disabled";
 	};
 
 	qos_isp: qos@fe8a0000 {
@@ -636,7 +649,9 @@
 					 <&cru HCLK_HOST_ARB>,
 					 <&cru SCLK_USB3_OTG0_REF>,
 					 <&cru SCLK_USB3_OTG0_SUSPEND>;
-				pm_qos = <&qos_pcie>;
+				pm_qos = <&qos_pcie>,
+					 <&qos_usb2>,
+					 <&qos_usb3>;
 			};
 			pd_vpu@RK1808_PD_VPU {
 				reg = <RK1808_PD_VPU>;
-- 
2.35.3

